src/breadcrumbs/breadcrumbs.scss

```diff
@@ -21,45 +21,94 @@
 
 @mixin iui-breadcrumbs-item {
   display: flex;
+  align-items: center;
   line-height: $iui-component-height;
   height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
 
-  a {
+  // #region Component overrides
+  > a {
     @include iui-anchor;
```

Comment:
Interesting...

![image](https://user-images.githubusercontent.com/81580355/163559916-a654723b-a045-4406-932e-a40f2e73db54.png)

![image](https://user-images.githubusercontent.com/81580355/163560018-ff87e139-263d-42ac-873f-36f85bbbc760.png)

Reply:
Yeah, these are coming from the anchor mixin.  I suppose we could split the anchor mixin up even more so we don't include the external modifier in this case.

Reply:
Updated `/anchor/` so `-external` is now in it's own mixin.